backup_and_restore/application_backup_and_restore/troubleshooting.adoc

```diff
@@ -28,6 +28,29 @@ include::modules/velero-obtaining-by-accessing-binary.adoc[leveloffset=+1]
 include::modules/oadp-debugging-oc-cli.adoc[leveloffset=+1]
 include::modules/migration-debugging-velero-resources.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../architecture/admission-plug-ins.adoc[Admission plug-ins]
+* xref:../../architecture/admission-plug-ins.adoc#admission-webhooks-about_admission-plug-ins[Webhook admission plug-ins]
+* xref:../../architecture/admission-plug-ins.adoc#admission-webhook-types_admission-plug-ins[Types of webhook admission plug-ins]
+
+[id="issues-with-velero-and-admission-workbooks"]
+== Issues with Velero and admission webhooks
+
+Velero has limited abilities to resolve admission webhook issues during a restore. If you have workloads with admission webhooks, you might need to use an additional Velero plug-in or make changes to how you restore the workload.
+
+Typically, workloads with admission webhooks require you to create a resource of a specific kind first.
+For example, if you restore or create a top-level object first, then the child resources are typically created already. Therefore, it is more likely that admission webhook problems are resolved because child resources, which are typically blocked by these admission webhooks, do not need to be created by Velero.
```

Comment:
I am not able to parse this line. Can you please clarify how this relates to the need to create a resource of a specific kind first?

Reply:
Creating a resource of a specific kind ie.. PizzaMaker automatically create child resources Pizza and DippingSauce which are sometimes has an admission webhook blocking creation without having a parent PizzaMaker created first.

Reply:
@bergerhoffer Can I use the link that Tiger suggests?

Reply:
@RichardHoch I think your changes helped with what Paul was pointing out. Restoring the top-level object service.serving.knative.dev illustrates the problem.